salt/utils/mount.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+'''
+Common functions for managing mounts
+'''
+
+# Import python libs
+from __future__ import absolute_import
+import logging
+import os
+import yaml
+
+# Import Salt libs
+import salt.utils  # Can be removed once is_true is moved
```

Comment:
It doesn't look like you're using any functions from `salt.utils` directly - can you remove this?

However, that does mean that you'll need to add `import salt.utils.stringutils`.